values.yaml

```diff
@@ -305,3 +305,35 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+
+
+## @section Realm creator parameters
+## @descriptionStart
+## This Helm chart provides the possibility to create a realm after Keycloak was deployed. This is done using [keycloak-config-cli](https://github.com/adorsys/keycloak-config-cli). All options from realmCreator.options are passed to this cli.
+## @descriptionEnd
+
+realmCreator:
+  ## @param realmCreator.enabled Enable the realm creator
+  enabled: false
+  image: 
+    ## @param realmCreator.image.repository The image for the realm creator
+    repository: adorsys/keycloak-config-cli
+    ## @param realmCreator.image.pullPolicy Pull policy of the image used for the realm creator
+    pullPolicy: IfNotPresent
+    ## @param realmCreator.image.tag The tag of the image for the realm creator
+    tag: 5.8.0-22.0.0
+    ## @param realmCreator.image.digest Overrides the image tag with a specific digest
+    digest: ""
+  ## @param realmCreator.username Specify the username to be used for the Keycloak API
+  username: admin
+  ## @param realmCreator.password Password for the Keycloak user
+  password: ""
+  ## @param realmCreator.existingSecret Use an existing secret for Keycloak admin password. This takes precedence over password.
+  existingSecret: ""
+  ## @param realmCreator.existingSecretKey
+  existingSecretKey: ""
+  ## @param realmCreator.options Specify how the realms should be created. It contains the JSON as YAML in a string. See [keycloak-config-cli](https://github.com/adorsys/keycloak-config-cli) for more information
+  options: ""
```

Comment:
just replace `options` with `realms` array and then check if empty?

Reply:
✅  https://github.com/intensiongmbh/keycloak-helm-chart/pull/3/commits/99340edcdd90efd4303b3170db6676b0d1d38b8b